lib/modules/manager/gleam/artifacts.ts

```diff
@@ -49,7 +49,7 @@ export async function updateArtifacts(
       ],
     };
 
-    await exec('gleam deps download', execOptions);
+    await exec('gleam deps update', execOptions);
```

Comment:
this will update all transitive deps to their latest version?

Reply:
yes, respecting compatible versions.